MeetU/MeetU/Views/Account/Login.cshtml

```diff
@@ -48,11 +48,18 @@
 @section Scripts {
     @Scripts.Render("~/bundles/jqueryval")
     <script>
-        $(".js-login-jump-icon").click(function () {
+        if ($(".validation-summary-errors").length > 0) {
```

Comment:
the class without 'js-' prefix shouldn't be used in js code. in this case, you should add a class called 'js-validation-summary-errors' to the element, then call it. The reason why we want to do that is we want to decouple logic and style. if in the future we want to change the html structure or just change some css, we can just change those classes without 'js-' prefix and don't need to have more concerns on logic.